### PR TITLE
Clarify deprecated API versions and releases

### PR DESCRIPTION
**Pull request by liggitt**
Clarify deprecated API versions and releases

/cc @deads2k @vllry

### CODE REVIEW

**Comment by liggitt** [comment]
This got referenced recently for the 1.16 release and there was some confusion about what was being removed and when (and the language around "None of these resources will be removed from Kubernetes or deprecated in any way.")

**Comment by liggitt** [comment]
cc @simplytunde
this would be good to get in ahead of the release as people look for guidance around the 1.16 changes

File: content/en/blog/_posts/2019-07-18-some-apis-are-being-deprecated.md [hunk 1]
```diff
@@ -10,32 +10,23 @@ slug: api-deprecations-in-1-16
...
+  * Migrate to use the **networking.k8s.io/v1** API, available since v1.8.
+    Existing persisted data can be retrieved/updated via the **networking.k8s.io/v1** API.
+* PodSecurityPolicy (in the **extensions/v1beta1** API group)
+  * Migrate to use the **policy/v1beta1** API, available since v1.10.
+    Existing persisted data can be retrieved/updated via the **policy/v1beta1** API.
+* DaemonSet, Deployment, StatefulSet, and ReplicaSet (in the **extensions/v1beta1** and **apps/v1beta2** API groups)
+  * Migrate to use the **apps/v1** API, available since v1.9.
+    Existing persisted data can be retrieved/updated via the **apps/v1** API.
+
+The **v1.19** release will stop serving the following deprecated API versions in favor of newer and more stable API versions:
```
Comment (simplytunde) [inline content/en/blog/_posts/2019-07-18-some-apis-are-being-deprecated.md]: @liggitt  is v1.19 the intended version here?

**Comment by simplytunde** [comment]
This lgtm once the comment is addressed.

File: content/en/blog/_posts/2019-07-18-some-apis-are-being-deprecated.md (hunk 1 above)
Comment (liggitt) [reply content/en/blog/_posts/2019-07-18-some-apis-are-being-deprecated.md]: ingress v1 work slipped to 1.17, so this actually got pushed out to 1.20. updated https://github.com/kubernetes/kubernetes/issues/43214 and this doc

**Pull request merged**